### PR TITLE
Adds schedule form wizard step names, and does some refactoring

### PR DESCRIPTION
**Pull request by AlexSCorey**
This addresses AAP-21595.  

It adds the step names for the wizard.  There is no save functionality, and all the steps show `Not implemented`.  Those steps are coming soon.

https://www.figma.com/proto/hDF6qJ0j8SQAF10nVGdwYo/Schedules?page-id=0%3A1&type=design&node-id=5-8674&viewport=244%2C550%2C0.02&t=mEc1kjnsd30gFGjN-1&scaling=scale-down

### CODE REVIEW

File: frontend/awx/views/schedules/wizard/ScheduleAddWizard.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  };
+
+  const onCancel = () => navigate(-1);
+
+  const { timeZones, links } = useGetTimezones();
+
+  const steps: PageWizardStep[] = [
+    {
+      id: 'typeStep',
+      label: t('Node details'),
```
Comment (matoval) [inline frontend/awx/views/schedules/wizard/ScheduleAddWizard.tsx:55]: According to the mockups this should be `Details`

File: frontend/awx/views/schedules/wizard/ScheduleAddWizard.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      inputs: <ScheduleInputs onError={() => {}} zoneLinks={links} timeZones={timeZones} />,
+    },
+    {
+      id: 'promptsStep',
+      label: t('Prompts'),
+      inputs: <PageNotImplemented />,
+    },
```
Comment (matoval) [inline frontend/awx/views/schedules/wizard/ScheduleAddWizard.tsx:62]: Missing Survey step

**Review by matoval — CHANGES_REQUESTED** [review]
Few small request

**Review by matoval — APPROVED** [review]

File: frontend/awx/views/schedules/wizard/ScheduleAddWizard.tsx (hunk 2 above)
Comment (AlexSCorey) [reply frontend/awx/views/schedules/wizard/ScheduleAddWizard.tsx:62]: Good catch

File: frontend/awx/views/schedules/wizard/ScheduleAddWizard.tsx (hunk 1 above)
Comment (AlexSCorey) [reply frontend/awx/views/schedules/wizard/ScheduleAddWizard.tsx]: Good catch

**Pull request merged**